CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Changelog
 
+<<<<<<< HEAD:CHANGELOG.rst
+1.18.6dev
+=========
+
+* doc-fix: Move workflow and PyTorch readme content into sphinx project
+* doc-fix: Fix broken links in using_tensorflow.rst
+* doc-fix: Update inference pipelines documentation
+* doc-fix: Move Chainer readme content into sphinx project
+
+1.18.5
+======
+=======
 ## 1.18.5
+>>>>>>> 840ba48edfe655eef71d90faac8644f8f7b53820:CHANGELOG.md
```

Comment:
can you remove these changes as well? I think you can do a `git checkout master -- CHANGELOG.md` to reset this file